test/token/ERC20/TokenTimelock.test.js

```diff
@@ -14,41 +14,55 @@ const TokenTimelock = artifacts.require('TokenTimelock');
 contract('TokenTimelock', function ([_, owner, beneficiary]) {
   const amount = new BigNumber(100);
 
-  beforeEach(async function () {
-    this.token = await MintableToken.new({ from: owner });
-    this.releaseTime = (await latestTime()) + duration.years(1);
-    this.timelock = await TokenTimelock.new(this.token.address, beneficiary, this.releaseTime);
-    await this.token.mint(this.timelock.address, amount, { from: owner });
-  });
+  context('with token', function () {
+    beforeEach(async function () {
+      this.token = await MintableToken.new({ from: owner });
+    });
 
-  it('cannot be released before time limit', async function () {
-    await expectThrow(this.timelock.release());
-  });
+    it('requires a future release time', async function () {
```

Comment:
How about "cannot be created with a release time in the past"? I like better something that describes what the test is doing, instead of what it attempts to prove